### DISCUSSION TITLE
Lesson 6: Foundry - Using Anvil Account to Interact with Sepolia

### DISCUSSION THREAD

**Discussion by spencerj411** [Cyfrin/foundry-full-course-cu#1383] [Q&A] [answered]
Hi! I was curious about whether I can use one of the local anvil accounts to deploy and/or interact with the Sepolia testnet node. I sent SepoliaETH from my main dev account (Account 1) to one of the anvil accounts. Account 1 lost SepoliaETH but the anvil account did not receive it. The anvil account still says 0 SepoliaETH even though it says 'Receive Confirmed' in the Activity tab (please see screenshot below). Could someone please help me understand why we can't use any of the local anvil accounts to deploy and/or interact with Sepolia? Many thanks

<img width="1127" alt="Screenshot 2024-02-06 at 12 33 42 am" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/24206502/ebde518f-ac80-45a8-9144-024e7cf8b0c4">

**Comment by heheboii11** [comment, marked as answer]
anvil blockchain is a local node it will not have any connection between the real ethereum nodes they just spin up a bunch of private keys and create certain address. You cannot directly interact with one blockchain to another, i.e, you cannot send crypto from one blockchain to another without using interportability protocols or bridges. I hope this information helps
